dev/devicelab/bin/tasks/build_ios_framework_module_test.dart

```diff
@@ -197,6 +197,16 @@ Future<void> _testBuildIosFramework(Directory projectDir, { bool isModule = fals
       'App.framework.dSYM'
     );
     checkDirectoryExists(appFrameworkDsymPath);
+
+    checkDirectoryNotExists(path.join(
+      outputPath,
+      mode,
+      'App.xcframework',
+      'ios-arm64',
+      'dSYMs',
+      'Flutter.framework.dSYM'
```

Comment:
Instead, test that the _only_ dSYM in this directory is `App.framework.dSYM` to catch the plugin dSYMs sneaking in, too.